assets/immersiverailroading/rolling_stock/freight/cp_logcar.json

```diff
@@ -16,7 +16,7 @@
 		"width": 1
 	},
 	    "freight": {
-        "slots": 1,
+        "slots": 2,
         "width": 1,
```

Comment:
Does width need to be increased for this to appear correctly? Or does it look fine?